src/hnsw/external_index.c

```diff
@@ -176,7 +179,7 @@ void StoreExternalIndex(Relation        index,
     wal_retriever_block_numbers = palloc0(sizeof(BlockNumber) * num_added_vectors);
```

Comment:
should this be `pfree`ed at the end of the function?

Reply:
Yes it is being `pfree` ed at the end of  the function [here](https://github.com/lanterndata/lanterndb/pull/26/files/67473f87e37e34709ba80f40bf99a7106d803fe6#diff-37df803808fb881daf0c4bda9979f2cb5ac24dc1f1ab34ca668ad2184dfc96eaR253)